helm/scylla/templates/serviceaccount.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if (and .Values.serviceAccount.create .Values.serviceAccount.annotations) -}}
```

Comment:
do we even need the create option at this point if we are only creating it when annotations are set and the reverse doesn't make sense?

Reply:
It doesn't make any sense but switch is already there, someone might be using it.

Reply:
it's a dead switch though